frontend/src/metabase/lib/analytics.js

```diff
@@ -26,12 +26,25 @@ export const trackPageView = url => {
  * @deprecated This uses GA which is not setup. We should use `trackSchemaEvent`.
  */
 export const trackStructEvent = (category, action, label, value) => {
+  /*
+  IMPORTANT NOTE: We don't track struct events via Google Analytics anymore.
+  This function only exists to prevent breaking changes.
+  There are still many references and tests that rely on this but their events will NOT be tracked anymore.
+  We plan to slowly remove this function and all references to it.
+  */
```

Comment:
**Reviewer note:** Added this note based on the discussion with @nemanjaglumac but we will likely remove it anyways

Reply:
This will make you happy :)
https://github.com/metabase/metabase/pull/46980